exporter/collector/integrationtest/testdata/fixtures/basic_counter_metrics_expect.json

```diff
@@ -600,74 +600,6 @@
       }
     ],
     "createMetricDescriptorRequests": [
-      {
-        "name": "projects/myproject",
```

Comment:
cc @dashpole (when you're back) these fixture updates look weird to me, not sure if this is expected or I removed too much code from here. either way can you ptal and confirm?

Reply:
See https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/342#discussion_r853158436.  The fixture changes here are the reverse of the fixture changes in the PR that added the GMP exporter, and exist because we no longer import exporterhelper.